File: javascript/space-age/space-age.js
export const age = (planet, seconds) => {
  let years = seconds / (60 * 60 * 24 * 365.25)

  switch(planet) {
    case 'mercury':
      years /= 0.2408467
      break
    case 'venus':
      years /= 0.61519726
      break
    case 'mars':
      years /= 1.8808158
      break
    case 'jupiter':
      years /= 11.862615
      break
    case 'saturn':
      years /= 29.447498
      break
    case 'uranus':
      years /= 84.016846
      break
    case 'neptune':
      years /= 164.79132
      break
  }

  return parseFloat(years.toFixed(2))
}

File: clojure/space-age/src/space_age.clj
(ns space-age)

(defn on-earth [seconds] ;; <- arglist goes here
  (/ seconds 31557600)
)

(defn on-mercury [seconds] ;; <- arglist goes here
  (/ (on-earth seconds) 0.2408467)
)

(defn on-venus [seconds] ;; <- arglist goes here
  (/ (on-earth seconds) 0.61519726)
)

(defn on-mars [seconds] ;; <- arglist goes here
  (/ (on-earth seconds) 1.8808158)
)

(defn on-jupiter [seconds] ;; <- arglist goes here
  (/ (on-earth seconds) 11.862615)
)

(defn on-saturn [seconds] ;; <- arglist goes here
  (/ (on-earth seconds) 29.447498)
)

(defn on-uranus [seconds] ;; <- arglist goes here
  (/ (on-earth seconds) 84.016846)
)

(defn on-neptune [seconds] ;; <- arglist goes here
  (/ (on-earth seconds) 164.79132)
)
